omnibus/config/software/openssl3.rb

```diff
@@ -97,7 +97,9 @@
   # the crazy platform specific compiler flags at the end.
   configure_args << env["CFLAGS"] << env["LDFLAGS"]
 
-  configure(*configure_args, bin: configure_cmd, env: env, no_build_triplet: true)
+  # We need to specify a bunch of extra options here since openssl's configure is not
+  # the usual autoconf configure but something handmade written in perl
+  configure(*configure_args, bin: configure_cmd, env: env, no_build_triplet: true, install_man: true)
```

Comment:
I wonder if it might even worth replacing this with a regular `command` to highlight how this is not a "normal `configure` script".

Reply:
done in https://github.com/DataDog/datadog-agent/pull/37321